server.js

```diff
@@ -7,26 +7,39 @@ require('dotenv').load();
 // Requires and constants if we need any
 const keystone = require('keystone');
 const http = require('http');
-const DB = require('./database.js');
-const uri = DB.uri;
+const env = keystone.get('env');
+
+function getDbUri() {
+	if (env === 'production') {
+		return require('./database.js').uri;
+	}
+	else if (env === 'development') {
+		return 'mongodb://localhost:27017/wusb';
+	}
+	else if (env === 'test') {
+		return 'mongodb://localhost:27017/wusb_test';
+	}
```

Comment:
what does this return if env doesn't equal one of the three - is it possible that it could? maybe have a case just incase that happens